modules/virt-about-monitored-resources-openshift-virtualization.adoc

```diff
@@ -0,0 +1,28 @@
+// Module included in the following assembly:
+//
+// * virt/logging_events_monitoring/virt-about-monitored-resources-openshift-virtualization.adoc
+//
+
+[id="virt-about-monitored-resources-openshift-virtualization_{context}"]
+= About monitored resources in {VirtProductName} Overview
+
+The following table shows actively monitored resources in the *{VirtProductName} Overview* page. This information is useful when you need obtain relevant data and intervene to resolve a problem.
```

Comment:
It is "Virtualization Overview" and I've updated the content to reflect this page name.

About relabeling from "resources" to "fields", here are some thoughts.
Each of the resources is presented on a "Card" in the page. Maybe consider:
"Cards with monitored resources" - would that be a possible title in the table?
In my conversations with the UX designer, he did not refer to these items as fields. He referred to them as cards.

![Screenshot from 2022-01-12 13-09-49](https://user-images.githubusercontent.com/55500825/149197847-3bb07f7a-fe01-40e8-9f13-862afaec3ee7.png)

![Screenshot from 2022-01-12 13-09-42](https://user-images.githubusercontent.com/55500825/149197873-5920a185-6e56-4f02-86a9-bae1ba7c9c56.png)

Reply:
@lmandavi I understand the terminology for the Top Consumers cards and am fine with referring to the CPU, Memory, etc. as resources. 
In my comment I was talking about the second screenshot shown here - Details, Activity, Permissions, Inventory, etc. These items are "fields" or "metrics" rather than resources and labeling them as resources might cause confusion because both tables in this assembly have similar titles and column headings.

Reply:
Thanks for your note. Since these items are a mix of fields and metrics, I will look into updating the table column heading.

Reply:
Please note - the .adoc file that will be updated is: virt-resources-monitored-actively-in-Virtualization-Overview-page.adoc.

Reply:
Renamed the table heading: Monitored resources, fields, and metrics.